testutil/src/main/java/org/spine3/test/NullToleranceTest.java

```diff
@@ -0,0 +1,418 @@
+            if (!preconditionClass) {
+                return false;
+            }
+
+            final StackTraceElement expectedUtilClassElement = stackTraceElements[1];
+            final boolean correct = isCorrectMethodName(method.getName(), expectedUtilClassElement.getMethodName());
+            if (!correct) {
+                return false;
+            }
+
+            final boolean correctClass = targetClassName.equals(expectedUtilClassElement.getClassName());
+            return correctClass;
+        }
+
+        private static boolean isCorrectMethodName(String expectedMethodName, String actualMethodName) {
+            final boolean correct = expectedMethodName.equals(actualMethodName);
+            return correct;
+        }
+    }
+
+    /**
+     * A builder for producing the {@link NullToleranceTest} instance.
+     */
+    public static class Builder {
+
+        private final Set<String> excludedMethods;
+        private final Map<? super Class, ? super Object> defaultValues;
+        private Class targetClass;
+
+        private Builder() {
+            defaultValues = newHashMap();
+            excludedMethods = newHashSet();
+        }
+
+        /**
+         * Sets the utility class.
+         *
+         * @param utilClass the utility {@link Class}
+         * @return the {@code Builder}
+         */
+        public Builder setClass(Class utilClass) {
+            this.targetClass = checkNotNull(utilClass);
+            return this;
+        }
+
+        /**
+         * Adds the method name which will be excluded during the check.
+         *
+         * @param methodName the name of the excluded method
+         * @return the {@code Builder}
+         */
+        @SuppressWarnings("WeakerAccess") // Will be used outside the package
+        public Builder excludeMethod(String methodName) {
+            checkNotNull(methodName);
+            final boolean validName = isName(methodName);
+            checkArgument(validName);
+
+            excludedMethods.add(methodName);
+            return this;
+        }
+
+        /**
+         * Adds the default value for the class which will be used during the check.
+         *
+         * @param value the default value for the class
+         * @return the {@code Builder}
+         */
+        @SuppressWarnings("WeakerAccess") // Will be used outside the package
+        public <I> Builder addDefaultValue(I value) {
+            checkNotNull(value);
+            defaultValues.put(value.getClass(), value);
+            return this;
+        }
+
+        /**
+         * Returns the target class.
+         *
+         * @return the {@link Class}
+         */
+        @VisibleForTesting
+        Class getTargetClass() {
+            return targetClass;
+        }
+
+        /**
+         * Return the {@code Set} of the excluded method name.
+         *
+         * @return the {@code Set}
```

Comment:
Please consult other `Builder`s in Spine on how to write the `@return...` parts. To my mind this tag can be omitted for getters like this, but please check.